src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -395,12 +454,28 @@ public synchronized Color getPixelColor(int x, int y) {
     }
 
     /**
-     * Creates an image containing pixels read from the screen.  This image does
-     * not include the mouse cursor.
+     * Creates an image containing pixels read from the screen.
+     * <p>
+     * If the desktop environment requires that permissions be granted
+     * to capture screen content, and the required permissions are not granted,
+     * then a {@code SecurityException} may be thrown,
+     * or the contents of the returned {@code BufferedImage} are undefined.
+     * <p>
+     * The {@link #revokeScreenCapturePermission()} can be used to revoke
+     * a previously granted permission.
+     * <p>
+     * @apiNote It is recommended to avoid calling this method on
+     * the AWT Event Dispatch Thread since screen capture may be a lengthy
+     * operation, particularly if acquiring permissions is needed and involves
+     * user interaction.
+     *
      * @param   screenRect      Rect to capture in screen coordinates
      * @return  The captured image
-     * @throws  IllegalArgumentException if {@code screenRect} width and height are not greater than zero
-     * @throws  SecurityException if {@code readDisplayPixels} permission is not granted
+     * @throws  IllegalArgumentException if {@code screenRect} width and height
+     *          are not greater than zero
+     * @throws  SecurityException if {@code readDisplayPixels} permission
+     *          is not granted, or user has not allowed any of his screens
+     *          to be captured.
```

Comment:
"user" -> "the user"
"his" -> gender neutral "their".
Or maybe better ", or access to the screen is denied by the desktop environment"

Reply:
Updated to the former for now.

Reply:
I think you should draft the CSR now based on the current text.
We may still need to tweak words like
"or the user has not allowed any of their screens to be captured."
based on developing understanding, or perhaps being careful not to be describe in the docs what happens to be the pipewire behaviour.
In fact that is why I suggested my other words, because (1) there may be no user involved and
(2) screen vs screens seems to be a pipewire / wayland concept. I'm pretty sure macOS just treats it as "the desktop" no matter how many monitors are involved.